src/Event/EventManager.php

```diff
@@ -442,6 +442,24 @@ public function prioritisedListeners($eventKey)
     }
 
     /**
+     * Returns the listeners matching a specified pattern
+     *
+     * @param string $eventKeyPattern Pattern to match.
+     * @return array
+     */
+    public function matchingListeners($eventKeyPattern)
+    {
+        $matchPattern = '/' . preg_quote($eventKeyPattern, "/") . '/';
```

Comment:
Yes, The resulting pattern would be something like `/^some\.event$/`. I feel that finding specific/exact events is a good place to start. If we start with patterns, we won't be able to take them back :smile:

How do you see more complex regex patterns being used?